configure

```diff
@@ -1042,7 +1042,7 @@ sub write_config_h {
   my %buildEnv = %{shift()};
   my $platform = $opts{$buildEnv{'build'}};
   my $pi = $platforminfo{$platform};
-  $opts{'optimize'} = 0 if (!exists $opts{'optimize'} || exists $opts{'sanitize'});
+  $opts{'optimize'} = 0 if (!(exists $opts{'optimize'} || exists $opts{'sanitize'}));
```

Comment:
```suggestion
  $opts{'optimize'} = 0 if exists $opts{'optimize'} && !exists $opts{'sanitize'};
```
OR
```suggestion
  $opts{'optimize'} = 0 if (exists $opts{'optimize'} && !exists $opts{'sanitize'});
```
I'm pretty sure parens are not needed here, but may be preferred anyway.

Reply:
`!(A || B) == (!A && !B)` so either way it needs to be `!exists $opt{'optimize'}` but I take your point about cleaning up the parentheses.
```suggestion
  $opts{'optimize'} = 0 if (!exists $opts{'optimize'} && !exists $opts{'sanitize'});
```